### PR TITLE
Support Proprietary Fonts from Design but optimize loading

### PR DESCRIPTION
**Pull request by andriyParashchuk**
Now we use the web-based fonts, because the addition of a custom font inline in css deteriorates the performance
Need add custom fonts after load page
The fonts are in `_assets/css/base/_fonts.scss`

### CODE REVIEW

**Comment by andriyParashchuk** [comment]
staging - http://jt-pr286-2.surge.sh/
@obliviusm @sweatC Pls check

**Comment by sweatC** [comment]
Font looks great
image
image

File: _assets/css/base/_general.scss [hunk 1]
```diff
@@ -68,6 +68,14 @@ body {
   background: $gray;
   min-width: $min-width;
 }
+.fonts-loaded body {
+  font-family: 'Graphik Web', Arial, sans-serif;
+  input,
+  textarea,
+  select {
+    font-family: 'Graphik Web', Arial, sans-serif;
```
Comment (sweatC) [inline _assets/css/base/_general.scss:76]: Why do you need to duplicate the same `font-family` here, if you have the same above?

**Review by sweatC — COMMENTED** [review]
@andriyParashchuk pls check

File: _assets/css/base/_general.scss (hunk 1 above)
Comment (andriyParashchuk) [reply _assets/css/base/_general.scss:76]: because these elements do not inherit font styles

Comment (sweatC) [reply _assets/css/base/_general.scss:76]: Accepted

**Review by sweatC — APPROVED** [review]

**Comment by pftg** [comment]
@andriyParashchuk could not find any difference:

**Comment by andriyParashchuk** [comment]
update staging - http://jt-pr286-2.surge.sh/

**Review by obliviusm — APPROVED** [review]
LGTM
good commit message

**Pull request merged**